llama-index-core/llama_index/core/extractors/metadata_extractors.py

```diff
@@ -56,7 +70,9 @@ class TitleExtractor(BaseExtractor):
     """
 
     is_text_node_only: bool = False  # can work for mixture of text and non-text nodes
-    llm: LLMPredictorType = Field(description="The LLM to use for generation.")
+    llm: Annotated[LLMPredictorType, WrapSerializer(add_class_name)] = Field(
```

Comment:
So for example here, when I call `TitleExtractor(...).model_dump()` will it still include the `llm` field?

Reply:
ahh that needs `SerializeAsAny` still. I have been annotating `LLM` and `BaseLLM` wherever I see it, but missed this one.

Are there are any parent classes not starting with `Base<..>` that we need to annotate with `SerializeAsAny`?

Reply:
Hmm, I can't think of any more off the top of my head, but I'm sure we will figure out edge cases as we go 😅

Reply:
yea i agree with fix as we go approach for the other cases that may come later! Going to push the changes covering `LLMPredictor` soon.

Reply:
for completeness of this thread: 

- this was addressed using `model_serializer` (see notes in PR message)